docs/docs/targeting.md

```diff
@@ -8,15 +8,33 @@ var modules = {
   }]
 };
 
-pathfora.initializeWidgets(modules, 'YOUR LYTICS ACCOUNT ID');
+// using the lytics callback assumes that window.liosetup exists and the lytics js tag is loaded after the pathfora config
+window.liosetup.callback = function(){ pathfora.initializeWidgets(modules, 'YOUR LYTICS ACCOUNT ID'); };
```

Comment:
Can we just remove the lytics account id here and in all the examples on this page? Since it's not required for this feature anymore we wouldn't want people to think it's required here.